widgets/layout-slider/styles/default.less

```diff
@@ -42,6 +43,12 @@
 
 			max-width: @slide_width;
 			height: @slide_height;
+			
+			& when (@vertically_align = true) {
+				display: flex;
+				flex-direction: column;
```

Comment:
Are there any layouts that you can think of that will break or have an issue due to `flex-direction: column;`. I can't demonstrate a case but am asking the question. Can you think of a problem here?

Reply:
I just ran a few tests with five existing SO Layout Slider widgets I had set up and didn't run into any issues. Can you please elaborate on what you're worried may happen? I'll see if there's any steps I can take to prevent that from happening.

Reply:
No worries, as long as we've had a go at thinking of possible issues and tested, we're ok for now.